prime-router/settings/staging/0174-md-phd-up-receiver-migration.yml

```diff
@@ -0,0 +1,104 @@
+# To submit this to localhost, run:
+#
+# ./prime multiple-settings set --input ./settings/staging/0174-md-phd-up-receiver-migration.yml
+#
+---
```

Comment:
This may need to move to ./settings/STLTs/MD/md-phd.yml as @GilmoreA6 did in his PR.